src/runtime-rs/crates/hypervisor/src/qemu/cmdline_generator.rs

```diff
@@ -1148,6 +1148,61 @@ impl ToQemuParams for Rtc {
     }
 }
 
+#[derive(Debug)]
+struct RngDevice {
+    // id is the device ID
+    id: String,
+
+    // filename is the entropy source on the host
+    filename: String,
+
+    // max_bytes is the bytes allowed to guest to get from the host’s entropy per period
+    max_bytes: u32,
+
+    // period is duration of a read period in seconds
+    period: u32,
+
+    // transport is the virtio transport for this device.
+    transport: String,
+}
+
+impl RngDevice {
+    fn new() -> RngDevice {
+        RngDevice {
+            id: "rng0".to_owned(),
+            filename: "/dev/urandom".to_owned(),
+            max_bytes: 1024,
+            period: 1000,
```

Comment:
If you look carefully at the go runtime, you'll see that `virtcontainers` doesn't set values for `MaxBytes` and `Period`. This causes `govmm` to ignore these settings and to generate a simple `-device virtio-rng-pci,rng=rng0`.

At this stage, we don't have any reason to take control on the default configuration of the device. Please drop `max_bytes` and `period` for now. We can still introduce them when we have a actual case for it.